app/src/main/java/Activities/BrowseProblemRecords.java

```diff
@@ -49,14 +49,13 @@ protected void onStart(){
         this.userId = intent.getStringExtra(USERIDEXTRA);
         this.problemUUID = intent.getStringExtra(PROBLEMIDEXTRA);
         try {
-            this.records = new ElasticsearchRecordController.GetRecordsWithProblemUUID().execute(this.problemUUID).get();
```

Comment:
there should also be an offline portion for getting records from offline file (used in case wifi is down)

Reply:
should put this in a controller